docs/operator/troubleshooting.md

```diff
@@ -0,0 +1,107 @@
+# Troubleshooting the Starboard Operator
+
+The Starboard Operator installs several Kubernetes resources into your Kubernetes cluster.
+
+Here are the common steps to check whether the operator is running correctly and to troubleshoot common issue.
+
+In addition to having a look at this section, you want to check [previous issues](https://github.com/aquasecurity/starboard/issues) to see if someone from the community had similar problems before.
+Feel free to either [open an issue](https://github.com/aquasecurity/starboard/issues), reach out on [Slack](https://slack.aquasec.com), or post your questions in the [discussion forum.](https://github.com/aquasecurity/starboard/discussions)
+
+## Installation
+
+Make sure that the latest version of the Starboard Operator is installed inside of your Kubernetes cluster.
+For this, have a look at the installation [options.](./installation)
```

Comment:
Actually we don't have docs/installation/index.md that describes different installation options. You'll get 404 - Not found error.

We can either link to particular installation option, e.g. ./installation/kubectl.md or create such overview page. Or remove this link altogether.

Reply:
but it links to the index.md within the operator, when I run the docs locally, it works? 
<img width="252" alt="Screenshot 2022-03-31 at 10 35 29" src="https://user-images.githubusercontent.com/33576047/161025216-af49a785-a40a-4a4a-9321-ad83a44fb365.png">

Reply:
Don't you see this warning in the logs:

```
INFO     -  Documentation built in 2.13 seconds
INFO     -  [09:37:54] Serving on http://0.0.0.0:8000/starboard/
WARNING  -  [09:37:59] "GET / HTTP/1.1" code 302
WARNING  -  [09:37:59] "GET /versions.json HTTP/1.1" code 404
INFO     -  [09:37:59] Browser connected: http://localhost:8000/starboard/
WARNING  -  [09:38:03] "GET /versions.json HTTP/1.1" code 404
INFO     -  [09:38:03] Browser connected: http://localhost:8000/starboard/operator/troubleshooting/
WARNING  -  [09:38:13] "GET /starboard/operator/troubleshooting/installation HTTP/1.1" code 404
WARNING  -  [09:38:13] "GET /versions.json HTTP/1.1" code 404
```

Anyway, if you intend to link to index.md use the .md suffix explicitly in MkDocs markdown.

Reply:
oh

Reply:
See https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages for more details:

![mkdocs_links](https://user-images.githubusercontent.com/1322923/161027519-68585559-dc58-4a9d-ae32-599323463fa3.png)